src/mito2/src/compaction.rs

```diff
@@ -818,6 +863,80 @@ mod tests {
     use crate::test_util::scheduler_util::{SchedulerEnv, VecScheduler};
     use crate::test_util::version_util::{VersionControlBuilder, apply_edit};
 
+    #[tokio::test]
+    async fn test_find_compaction_options_db_level() {
```

Comment:
nit: we need to four test cases to cover all possible setup:
- db level compaction options set, table set (override)
- db level compaction options set, table unset
- db level compaction options unset, table set (override)
- db level compaction options unset, table unset

Reply:
done， thank you very much.